### PR TITLE
Add maintenance section to trigger cron jobs

### PR DESCRIPTION
**Pull request by joschahenningsen**
### Motivation and Context

Sometimes cronjobs need to be triggered manually, e.g. when we add a new feature that fetches data occasionally but the cron is scheduled in the future after deployment

### Description

This adds a section to the maintenance page to trigger any cron job registered.

### Steps for Testing

Prerequisites:
- 1 Admin

1. Log in
2. Navigate to maintenance page
3. Test if running a job triggers the corresponding function

### Screenshots

(dammit, obs didn't catch the overlay.)

https://user-images.githubusercontent.com/44805696/205509525-803c18b2-c1d6-4f59-89eb-0031c6e57f77.mp4

### CODE REVIEW

File: api/maintenance.go [hunk 1]
```diff
@@ -81,3 +87,22 @@
...
+	if err != nil {
+		_ = c.Error(tools.RequestError{
+			Status:        http.StatusBadRequest,
+			CustomMessage: "Can't read request",
+			Err:           err,
+		})
+		return
+	}
+	jobName := c.Request.FormValue("job")
+	log.Info("request to run ", jobName)
```
[the remark this answers is not in the document]
Comment (joschahenningsen) [reply api/maintenance.go:106]: Dismissed as false positive.

File: web/template/admin/admin_tabs/maintenance.gohtml [hunk 2]
```diff
@@ -1,18 +1,41 @@
...
-    <div class="form-container" x-data="admin.maintenancePage()" x-init="keepUpdated()">
-        <div class="form-container-title">Thumbnails</div>
-        <div class="form-container-body" x-init="setTimeout(() => {console.log('hi')}, 200)">
-            <button @click="generateThumbnails()" class="btn" >Regenerate All Thumbnails</button>
-            <div class="p-4" x-show="running">
-                <span class="text-1 font-semibold text-sm">Progress:</span>
+        <h2 class="text-2xl text-1 my-auto mt-4 font-medium">Maintenance</h2>
+        <div class="form-container" x-init="keepUpdated()">
+            <div class="form-container-title">Thumbnails</div>
+            <div class="form-container-body" x-init="setTimeout(() => {console.log('hi')}, 200)">
```
Comment (MatthiasReumann) [inline web/template/admin/admin_tabs/maintenance.gohtml]:
```suggestion
            <div class="form-container-body">
```

File: web/template/admin/admin_tabs/maintenance.gohtml [hunk 3]
```diff
@@ -1,18 +1,41 @@
...
+                <button @click="generateThumbnails()" class="btn">Regenerate All Thumbnails</button>
+                <div class="p-4" x-show="running">
+                    <span class="text-1 font-semibold text-sm">Progress:</span>
                     <div class="w-full ml-2">
                         <div class="font-semibold text-sm text-4" x-text="`${Math.floor(process*100)}%`"></div>
                         <div class="w-full bg-gray-200 rounded-full h-1.5 mb-1 dark:bg-gray-700">
-                            <div class="bg-blue-600 h-1.5 rounded-full dark:bg-blue-500" :style="`width: ${process*100}%`"></div>
+                            <div class="bg-blue-600 h-1.5 rounded-full dark:bg-blue-500"
+                                 :style="`width: ${process*100}%`"></div>
```
Comment (MatthiasReumann) [inline web/template/admin/admin_tabs/maintenance.gohtml]: Sorry for nitpicking - `process` should actually be named `progress` right? 😅

**Review by MatthiasReumann — CHANGES_REQUESTED** [review]
Looks great 💯

I left some tiny remarks but other than that I think this PR is good to go.

File: web/template/admin/admin_tabs/maintenance.gohtml (hunk 3 above)
Comment (joschahenningsen) [reply web/template/admin/admin_tabs/maintenance.gohtml]: Sure, fixed that, thanks :)

File: web/template/admin/admin.gohtml [hunk 4]
```diff
@@ -82,7 +82,7 @@
                             <a class="px-3 py-2 transition-colors duration-200 {{if eq $page "maintenance"}}text-1{{else}}text-5{{end}} relative block"
                                href="/admin/maintenance"><span
                                         class="rounded-md absolute inset-0 bg-cyan-50 opacity-0"></span><span
-                                        class="relative">Aaintenance</span></a></li>
+                                        class="relative">Maintenance</span></a></li>
```
Comment (MatthiasReumann) [inline web/template/admin/admin.gohtml:85]: Well that slipped by me 🫠

**Review by MatthiasReumann — APPROVED** [review]
LGTM 💯

File: web/template/admin/admin.gohtml (hunk 4 above)
Comment (joschahenningsen) [reply web/template/admin/admin.gohtml:85]: By me as well 😀

**Pull request merged**